forte/data/data_store.py

```diff
@@ -187,6 +197,76 @@ def _new_tid(self) -> int:
         r"""This function generates a new ``tid`` for an entry."""
         return uuid.uuid4().int
 
+    def _get_type_info(self, type_name: str) -> Dict[str, int]:
+        """
+        Add a new type input_entry_class into ``_type_attributes``.
+        This method will get all attributes of a type and assign index to them.
```

Comment:
Similar to above, when describing the function, `assign index` is not important. You can add a second paragraph providing some implementation details, but if you want to do that, it needs to be done in much more details. For example, "assign index" is to vague